app/views/todos/_pull_requests.html.erb

```diff
@@ -0,0 +1,26 @@
+<div>
+  <h3>Pull Requests (<%= @my_pulls.size %>): </h3>
+  <table class="table">
+    <thead>
+      <tr class="table__row table__row--head">
+        <th>Number</th>
+        <th>Title</th>
+        <th>State</th>
+        <th>Created</th>
+      </tr>
+    </thead>
+    <tbody>
+    <% @my_pulls.each do | pull | %>
+      <tr class="table__row">
+        <td><%= pull.pull_request_number %></td>
+        <td>
+          <%= link_to pull.repository_name, pull.repository_link, target: :_blank, class: "project_name"%><br/>
+          <%= link_to pull.title, pull.pull_request_link, target: :_blank %>
+        </td>
+        <td><%= pull.state %></td>
+        <td><%= created_at(pull) %></td>
+      </tr>
+    <% end %>
+    </tbody>
+  </table>
+</div>
```

Comment:
@cleicar Isn't there an opportunity here to refactor and DRY the code between pull request and issue views? I see that they are 90% similar...